README.md

```diff
@@ -1,54 +1,83 @@
-# OHBM2020 posters Jitsi rooms
+# :door::video_camera: OHBM2020 Posters Jitsi Rooms
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
 [![All Contributors](https://img.shields.io/badge/all_contributors-6-orange.svg?style=flat-square)](#contributors-)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
 
-Just go to https://datalad-datasets.github.io/ohbm2020-posters/ . Wait for the page to load.
-Search for a poster by entering desired words present in the title name or presenter name or keywords and table will
-limit to the results of the search.  Click on the poster you would like to "present" or "attend", and get into a
-dedicated jitsi room.
-
-Unfortunately "source" spreadsheet neither had presentation dates/times, nor URLs to PDFs.  So we might improve upon
-that (send a PR if you see how).
-
-## Motivation
-
-As much as we had fun preparing our [Center of Open Neuroscience: DataLad, ReproNim, et al](https://ohbm.6connex.com/event/OHBMAnnualMeeting/en-us#!/CenterforOpenNeuroscience) booth, experience with the conference platform quickly
-showed its shortcomings: poster sessions seems will be limited to "open PDF, find presenter somewhere in an old
-fashion chat and 'converse' there". It is not even 2010 -- it is more of 2000.  There were discussions about
-establishing some gather.town room for poster presentations etc.  
-
-Since there is a spreadsheet with all the posters, and unique IDs for each one (but unfortunately without PDF URLs
-to be added to git-annex/DataLad), we decided to (ab)use Jitsi to provide each poster a dedicated video
- room.
+## :question: Motivation
+<p align=justify>As much as we had fun preparing our <a href="https://ohbm.6connex.com/event/OHBMAnnualMeeting/en-us#!/CenterforOpenNeuroscience">Center of Open Neuroscience: DataLad, ReproNim, et al.</a> booth, experience with the conference platform quickly showed its shortcomings: poster sessions seemed to be limited to "open a PDF, find the presenter somewhere in an old fashioned chat and 'converse' there". It is not even 2010 -- it is more of 2000. There were discussions about
```

Comment:
Ideally all replaced with HTML markdown for URLs should come back, i.e instead of `<a href="URL">TEXT</a>` it should become `[TEXT](URL)`